content/docs/command-reference/params/index.md

```diff
@@ -165,6 +168,50 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Print all parameters
+
+Following the previous example, we can use `dvc params diff` to list all of the
+param values available in the <abbr>workspace</abbr>:
+
+```dvc
+$ dvc params diff
+Path         Param           Old    New
+params.yaml  lr              —      0.0041
+params.yaml  process.bow     —      15000
+params.yaml  process.thresh  —      0.98
+params.yaml  train.epochs    —      70
+params.yaml  train.layers    —      9
+```
+
+This command shows the difference in parameters between the workspace and the
+last committed version of the `params.yaml` file. In our example there's no
+previous version, which is why all `Old` values are `—`.
+
+## Example: Parameterize stages in `dvc.yaml`
```

Comment:
To be honest, the whole `dvc params` is more about parameter dependencies, not sure this example is relevant at all.

Reply:
Example removed. The note in the description now links to the section in dvc.yaml